extensions/README.md

```diff
@@ -6,51 +6,106 @@ AMP Extensions are either extended components or extended templates.
+
+| Component | Description |
+| --------- | ----------- |
+| [`amp-form`](amp-form/amp-form.md) | Provides form support. |
+
+### Frames 
+
+| Component | Description |
+| --------- | ----------- |
+| [`amp-iframe`](amp-iframe/amp-iframe.md) | Displays an iframe. |
+
+### Presentation
+
+| Component | Description |
+| --------- | ----------- |
+| [`amp-accordion`](amp-accordion/amp-accordion.md) | Provides a way for viewers to have a glance at the outline of the content and jump to a section of their choice at will. |
+| [`amp-anim`](amp-anim/amp-anim.md) | Manages an animated image, typically a GIF. |
+| [`amp-carousel`](amp-carousel/amp-carousel.md) | Displays multiple similar pieces of content along a horizontal axis. |
 | [`amp-dynamic-css-classes`](amp-dynamic-css-classes/amp-dynamic-css-classes.md) | Adds several dynamic CSS class names onto the HTML element. |
-| [`amp-facebook`](amp-facebook/amp-facebook.md) | Displays a Facebook post or video. |
 | [`amp-fit-text`](amp-fit-text/amp-fit-text.md) | Expands or shrinks font size to fit the content within the space given. |
 | [`amp-font`](amp-font/amp-font.md) | Triggers and monitors the loading of custom fonts. |
 | [`amp-fx-flying-carpet`](amp-fx-flying-carpet/amp-fx-flying-carpet.md) | Wraps its children in a unique full-screen scrolling container allowing you to display a full-screen ad without taking up the entire viewport. |
-| [`amp-iframe`](amp-iframe/amp-iframe.md) | Displays an iframe. |
 | [`amp-image-lightbox`](amp-image-lightbox/amp-image-lightbox.md) | Allows for an “image lightbox” or similar experience. |
-| [`amp-instagram`](amp-instagram/amp-instagram.md) | Displays an Instagram embed. |
-| [`amp-install-serviceworker`](amp-install-serviceworker/amp-install-serviceworker.md) | Installs a ServiceWorker. |
-| [`amp-jwplayer`](amp-jwplayer/amp-jwplayer.md) | Displays a cloud-hosted [JW Player](https://www.jwplayer.com/). |
-| [`amp-kaltura-player`](amp-kaltura-player/amp-kaltura-player.md) | Displays the Kaltura Player as used in [Kaltura's Video Platform](https://corp.kaltura.com/). |
 | [`amp-lightbox`](amp-lightbox/amp-lightbox.md) | Allows for a “lightbox” or similar experience. |
-| [`amp-list`](amp-list/amp-list.md) | Dynamically downloads data and creates list items using a template. |
 | [`amp-mustache`](amp-mustache/amp-mustache.md) | Allows rendering of [`Mustache.js`](https://github.com/janl/mustache.js/) templates. |
-| [`amp-pinterest`](amp-pinterest/amp-pinterest.md) | Displays a Pinterest widget or Pin It button. |
-| [`amp-reach-player`](amp-reach-player/amp-reach-player.md) | Displays a [Beachfront Reach](https://beachfrontreach.com/) video player. |
 | [`amp-sidebar`](amp-sidebar/amp-sidebar.md) | Provides a way to display meta content intended for temporary access such as navigation, links, buttons, menus. |
+| [`amp-user-notification`](amp-user-notification/amp-user-notification.md) | Displays a dismissable notification to the user. |
+
+### Scripts
```

Comment:
maybe we can have a more descriptive section for the service worker stuff?

cc @ericlindley-g